mod/qbassign/externallib.php

```diff
@@ -3573,27 +3572,28 @@ public static function create_assignment_service($courseid,$siteid,$chapterid,$t
                      $onlinetext_default = $DB->insert_record('qbassign_plugin_config', $updatesactivityonline);
                  }           
              }
-             purge_all_caches();
              $DB->set_field('qbassign', 'uid', $uid, array('id' => $returnid));
              $lti_updated = [                        
                              'message'=>'Success here',
```

Comment:
Just give the text "Successfully created a assignment or Successfully updated a assignment"